project-fortis-interfaces/src/components/Insights/Dashboard.js

```diff
@@ -167,13 +172,16 @@ export default class Dashboard extends React.Component {
     );
   }
 
-  refreshDashboard = (includeCsv) => {
+  refreshDashboard = (includeCsv, replaceBbox) => {
     const { dataSource, timespanType, termFilters, datetimeSelection, zoomLevel, maintopic, bbox, fromDate, toDate, externalsourceid, selectedplace } = this.filterLiterals();
-    this.props.flux.actions.DASHBOARD.reloadVisualizationState(fromDate, toDate, datetimeSelection, timespanType, dataSource, maintopic, bbox, zoomLevel, Array.from(termFilters), externalsourceid, includeCsv, selectedplace);
+    if (replaceBbox === null) {
```

Comment:
Nit: you can inline this using this standard pattern for default values: `replaceBbox = replaceBbox || bbox`